Package.swift

```diff
@@ -143,5 +38,351 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-collections.git", from: "1.0.2"),
         .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0"),
     ],
-    targets: targets
+    targets: [
```

Comment:
I think it would be useful to group targets into 'blocks', e.g. "modules", "examples", and "tests" and separate each with a `// MARK:` or similar.

Reply:
Added `// MARK:`s and grouped them